src/stirling/protocol_inference/dataset_generation.py

```diff
@@ -86,7 +86,8 @@ def gen_tshark_cmd():
     -e tcp.srcport \
     -e udp.srcport \
     -e tcp.dstport \
-    -e udp.dstport"
+    -e udp.dstport \
+    -d tcp.port==27017,mongo"
 
```

Comment:
I can't easily comment on the connection matrix, so adding my comment here. Can you explain again why we have 0.9% of Mongo showing up as unknown in latest matrix?

Reply:
One more detail I forgot, it would be helpful to also explain why the packet counts are lower in the new matrix (that we were double, triple, 4x counting things).